wfdb/plot/plot.py

```diff
@@ -217,11 +216,11 @@ def plot_items(
     sampling_freq : number or sequence, optional
         The sampling frequency or frequencies of the signals.  If this is a
         list, it must have the same length as the number of channels.  If
-        unspecified, defaults to `fs`.
+        unspecified, defaults to `frame_freq`.
```

Comment:
Having these two parameters is rather confusing. Can you explain why both `sampling_freq` and `frame_freq` are both needed?